src/main/java/com/google/cloud/pontem/SpannerUtil.java

```diff
@@ -61,6 +67,71 @@
     return options;
   }
 
+  /**
+   * Convert the Cloud Spanner type from {@type String} to {@type Type}
+   *
+   * @see /google-cloud-java/google-cloud-clients/apidocs/com/google/cloud/spanner/Type.Code.html
+   */
+  public static Type getSpannerType(String typeAsString) {
+    typeAsString = typeAsString.trim();
+    if (typeAsString.startsWith("STRING")) {
+      return Type.string();
+    } else if (typeAsString.startsWith("INT64")) {
+      return Type.int64();
+    } else if (typeAsString.startsWith("DATE")) {
+      return Type.date();
+    } else if (typeAsString.startsWith("BOOL")) {
+      return Type.bool();
+    } else if (typeAsString.startsWith("FLOAT64")) {
+      return Type.float64();
+    } else if (typeAsString.startsWith("BYTES")) {
+      return Type.bytes();
+    } else if (typeAsString.startsWith("TIMESTAMP")) {
+      return Type.timestamp();
+    } else if (typeAsString.startsWith("ARRAY<") && typeAsString.endsWith(">")) {
+      // parse out the specific column statements
+      Pattern compiledPattern = Pattern.compile("^ARRAY<([a-zA-Z0-9()]+)>$", Pattern.DOTALL);
```

Comment:
Is it better to keep the pattern as a private static field so that it's compiled only once upon initialization?